package.json

```diff
@@ -2,7 +2,7 @@
     "name": "tsiclient",
     "main": "tsiclient.js",
     "description": "",
-    "version": "1.4.13",
+    "version": "1.4.17",
```

Comment:
Why is this bumped by 4?

Reply:
seems 1.4.16 is the latest published version here: https://www.npmjs.com/package/tsiclient